sdk/monitor/monitor-opentelemetry-exporter/src/utils/constants/applicationinsights.ts

```diff
@@ -31,7 +31,7 @@ export enum DependencyTypes {
   Wcf = "WCF Service",
 }
 
-export const AzureMonitorSampleRate = "_MS.sampleRate";
+export const AzureMonitorSampleRate = "microsoft.sample_rate";
 export const ApplicationInsightsBaseType = "_MS.baseType";
```

Comment:
It would be nice to align and remove all "_MS" attributes we are using

Reply:
Doesn't look like the last two I see in this file have corresponding specs to update them, but I'll make a work item to discuss updating these with breeze.

Reply:
Those are not Breeze specific, they are internal in our code, we were using different name standardization, perfect time to align now that we have at least one internal attribute defined in spec